h/templates/admin/groups.html.jinja2

```diff
@@ -40,6 +41,10 @@
                 {{ group_url }}
               </a>
             </td>
+            <td>
+              {% set org_url = request.route_url('admin_organizations_edit', pubid=group.organization.pubid) %}
```

Comment:
It took me a few experimental clicks in the interface to ascertain that the name column linked to the edit tool, though I don't know if there's any better way to surface that...

Reply:
Just some nice <samp>Edit</samp> buttons next to the names?

Reply:
I had no problem with it.